From ca96c4c58042576c57c760aef6884f30a07e713d Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Sun, 8 Jan 2012 16:21:53 -0500
Subject: [PATCH] Always build cairo-gobject gir, even if cairo is not
 installed

This is needed to avoid circular dependencies with cairo and splitting
gobject-introspection-cairo out of the main package (the patch to do so
would be invasive and hard to maintain).

https://bugs.gentoo.org/show_bug.cgi?id=391213
---
 configure.ac |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/configure.ac b/configure.ac
index cbee79e..5bc36f7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -141,6 +141,7 @@ if test x$have_cairo = xyes; then
   PKG_CHECK_MODULES(CAIRO_GOBJECT, [cairo-gobject], have_cairo_gobject=yes, have_cairo_gobject=no)
 fi
 
+have_cairo_gobject=yes
 if test x$have_cairo_gobject = xyes; then
   case "$host" in
     *-*-darwin*)
-- 
1.7.8.3

